sql/parser/eval_test.go

```diff
@@ -212,9 +212,9 @@ func TestEvalExprError(t *testing.T) {
 		{`1.0 AND true`, `cannot convert float to bool`},
 		{`'a' AND true`, `cannot convert string to bool`},
 		{`(1, 2) AND true`, `cannot convert tuple to bool`},
-		{`lower()`, `incorrect number of arguments`},
-		{`lower(1, 2)`, `incorrect number of arguments`},
-		{`lower(1)`, `argument type mismatch`},
+		{`lower()`, `unknown function lower()`},
```

Comment:
The error is an artifact of their implementation. I would argue it is not very informative. The `length` function certainly exists, you just used the incorrect number of arguments.